src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -232,6 +232,7 @@ private static HttpMiddleware BuildMiddleware(DocsEnvironment? environment = nul
                         {
                             DocsEnvironment.Prod => s_opsTokenPublic,
                             DocsEnvironment.PPE => s_opsTokenPubDev,
+                            DocsEnvironment.Perf => s_opsTokenPubDev,
```

Comment:
Does this need to be a separate source environment?

Reply:
Updated.